buildsys-pre-build

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+set -ex
+
+SPEC_NAME=cloudlinux-nsqd
+PKG_VERSION=$(awk '$1 == "Version:" {print $2}' < "${SPEC_NAME}.spec")
+PKG_NAME=$(awk '$1 == "Name:" {print $2}' < "${SPEC_NAME}.spec")
+
+if ! VERSION=$(git describe --tags || git rev-parse HEAD); then
+  VERSION="0"
+fi
+echo $VERSION > build_version
+
+ARCHIVE_NAME=$PKG_NAME-$PKG_VERSION.tar.gz
+
+tar \
+  --create \
+  --auto-compress \
+  --file /tmp/$ARCHIVE_NAME \
+  --exclude .git \
+  --exclude $ARCHIVE_NAME \
+  --transform "s#^\./#$PKG_NAME-$PKG_VERSION/#" \
+  ./
+
+mv /tmp/$ARCHIVE_NAME .
```

Comment:
Are you sure this is necessary? Build-system can make git archive by itself, but buildsys-pre-build takes a lot of time

Reply:
It's the way unified-access-logger do it, there are imunify-common which has kind of similar script.

Reply:
Maybe they do not know how the build system works and @AvoidMe knows it? :)

Reply:
defence360 works in pretty similar fashion :)